libs/utils/test.py

```diff
@@ -42,20 +42,31 @@ def _init(cls, conf, *args, **kwargs):
             cls.logger.setLevel(kwargs['loglevel'])
             kwargs.pop('loglevel')
 
-        cls.conf = conf
-
         cls._runExperiments()
 
     @classmethod
+    def _getTestConf(cls):
+        if not hasattr(cls, "test_conf"):
```

Comment:
Here is not clear that the "test_conf" is expected to be defined by one of the derived classes.

Why not adding a NoneType initialized `cls.test_conf` attribute and check here that the user has overridden its definition to be a dictionary?

Dose it make sense to use this method also to provide a sanity check of the provided configuration dictionary?